### ISSUE TITLE
Non-training of data

### ISSUE THREAD

**Issue by jenny-pixel** [prabhakar-sivanesan/Custom-keypoint-detection#1]
Hey, thanks so much for this tutorial. This is much less of an issue with your tutorial and more an issue with the tf model library I expect!

Basically, when I follow all your steps, during training I get 

keypoint/offset': 0.0,
regression': 0.0,

and a total loss rate that very quickly converges to 0. Needless to say, the resulting model is useless.

Any ideas about what might be happening? Any ideas would be much appreciated!

Cheers

**Comment by prabhakar-sivanesan** [comment]
Hi, Please provide adequate information like total images in the dataset, your training configuration and some training logs. This will help me determine the problem better.

**Comment by bnavras** [comment]
Hi, I have the same problem
[configs](https://dropmefiles.com/oii9M)
dataset: 77 train images, 34 test images
![image](https://user-images.githubusercontent.com/17747968/195255758-98a2fee0-53eb-40ff-9d70-941db23815e2.png)

**Comment by bnavras** [comment]
@jenny-pixel have you solved the problem?

**Comment by jenny-pixel** [comment]
Hey, no I didn't annoyingly
________________________________

**Comment by Shubham654** [comment]
Got the same issue! 
` 'Loss/doc_scanner/keypoint/offset': 0.0,
 'Loss/doc_scanner/keypoint/regression': 0.0,`

any update on this? @jenny-pixel @prabhakar-sivanesan

**Comment by Shubham654** [comment]
@jenny-pixel Did you fixed the issue from your side???

**Comment by Shubham654** [comment]
`I1201 10:29:51.477410 139825162438528 model_lib_v2.py:708] {'Loss/box/offset': 0.058085866,
 'Loss/box/scale': 0.6349598,
 'Loss/doc_scanner/keypoint/heatmap': 0.00023415813,
 'Loss/doc_scanner/keypoint/offset': 0.2739461,
 'Loss/doc_scanner/keypoint/regression': 0.36031744,
 'Loss/object_center': 0.0025841102,
 'Loss/total_loss': 1.3301275,
 'learning_rate': 0.00077874475}`
 
 **Update:** Solved the Issue by rechecking the config file...
 
 The Issue was in the class name where I mismatched the name from the labels file and config... It needs to be the same.

**Comment by jenny-pixel** [comment]
Excellent, thank you!
________________________________

**Comment by prabhakar-sivanesan** [comment]
Closing as this issue is solved.
